junit5/src/main/java/cz/xtf/junit5/extensions/OpenShiftRecorderHandler.java

```diff
@@ -59,12 +63,25 @@ public void beforeEach(ExtensionContext context) {
 
     @Override
     public void handleTestExecutionException(ExtensionContext context, Throwable throwable) throws Throwable {
+        Throwable newThrowable = throwable;
+        final ByteArrayOutputStream messageStream = new ByteArrayOutputStream();
         try {
             openShiftRecorderService.recordState(context);
+
+            try (PrintStream ps = new PrintStream(messageStream, true, StandardCharsets.UTF_8.name())) {
+                openShiftRecorderService.printPodLogs(context, ps);
+                openShiftRecorderService.printEvents(context, ps);
+                openShiftRecorderService.printBuildLogs(context, ps);
+            }
+            if (messageStream.size() != 0) {
+                Constructor<? extends Throwable> constructor = throwable.getClass().getConstructor(String.class,
+                        Throwable.class);
+                newThrowable = constructor.newInstance(throwable.getMessage() + messageStream, throwable.getCause());
```

Comment:
I think that storing all logs into `ByteArrayOutputStream messageStream` is dangerous as it might cause OutOfMemoryError when logs are too large. Is there a way to stream the content into the exception message?